app/assets/javascripts/discourse/tests/acceptance/flag-post-test.js

```diff
@@ -129,6 +130,14 @@ acceptance("flagging", function (needs) {
     assert.ok(!exists(".modal-body"));
   });
 
+  test("Can delete spammer from spam", async function (assert) {
+    await visit("/t/internationalization-localization/280");
+    await openFlagModal();
+    await click("#radio_spam");
+
+    assert.ok(exists(".delete_spammer"));
```

Comment:
personal preference :p

```suggestion
    assert.dom(".delete_spammer").exists();
```

Reply:
I will leave this as it is for now, since the surrounding examples use the`exists` helper, but maybe we can retire the helper in a cleanup PR now if there's a native alternative? 🙂

Reply:
yes sure :+1: